pyproject.toml

```diff
@@ -15,6 +15,7 @@ aiohttp = "^3.11.13"
 asyncio = "^3.4.3"
 
 [tool.poetry.dev-dependencies]
+ansible-lint = "*"
```

Comment:
You need to regenerate a lock file after this change:
```
pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.
```